Build/Common.Build.Default.props

```diff
@@ -29,8 +29,9 @@
 
   <!-- Common build settings -->
   <PropertyGroup>
-    <OptimizedBuild />
-    <OptimizedBuild Condition="'$(Configuration)'=='Release' or '$(Configuration)'=='Test'">true</OptimizedBuild>
+    <FreeBuild />
+    <FreeBuild Condition="'$(Configuration)'=='Release' or '$(Configuration)'=='Test'">true</FreeBuild>
```

Comment:
I think having a bool property is cleaner: if we ever introduce another variant of a "debug build", this will work better.